src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    public static final int MAGIC_NUMBER = 47;
+    private static final Random random = new Random(MAGIC_NUMBER);
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)].name();
     }
 }
```

Comment:
According to the checklist, static methods are generally a bad practice. Instead of providing a static method 'getRandomColor', instantiate the 'ColorSupplier' class where you need to use it.